hack/tools/kustomize/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: build
+
+KUSTOMIZE_VERSION ?= 2.0.0
+IMAGE_NAME ?= akutz/kustomize
```

Comment:
Fix this in follow-up or now?

Reply:
Later. I’m waiting on a new GCS project.